### PR TITLE
Events redesign

### PR DESCRIPTION
**Pull request by BryanRumsey**

### CODE REVIEW

File: client/views/edit-event-assignment.js [hunk 1]
```diff
@@ -62,44 +53,66 @@ module.exports = View.extend({
   getOptions: function () {
     var species = this.model.collection.parent.collection.parent.species;
     var parameters = this.model.collection.parent.collection.parent.parameters;
-    var speciesNames = species.map(function (specie) { return specie.name });
-    var parameterNames = parameters.map(function (parameter) { return parameter.name });
-    return speciesNames.concat(parameterNames);
+    var specs = species.map(function (specie) {
```
Comment (jtcooper10) [inline client/views/edit-event-assignment.js:56]: Consider removing the "Variable" or "Parameter" options groups when there are no available variables/parameters

**Review by jtcooper10 — CHANGES_REQUESTED** [review]

File: client/templates/includes/eventAssignmentsEditor.pug [hunk 2]
```diff
@@ -1,26 +1,55 @@
...
 
-            div.tooltip-icon(data-html="true" data-toggle="tooltip" title=this.parent.parent.tooltips.variable) <sup><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="info" class="svg-inline--fa fa-info fa-w-6" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 192 512"><path fill="currentColor" d="M20 424.229h20V279.771H20c-11.046 0-20-8.954-20-20V212c0-11.046 8.954-20 20-20h112c11.046 0 20 8.954 20 20v212.229h20c11.046 0 20 8.954 20 20V492c0 11.046-8.954 20-20 20H20c-11.046 0-20-8.954-20-20v-47.771c0-11.046 8.954-20 20-20zM96 0C56.235 0 24 32.235 24 72s32.235 72 72 72 72-32.235 72-72S135.764 0 96 0z"></path></svg></sup>
+    button.btn.btn-outline-collapse(data-toggle="collapse" data-target="#collapse-event-assignments" data-hook="collapse") -
```
[the remark this answers is not in the document]
Comment (BryanRumsey) [reply client/templates/includes/eventAssignmentsEditor.pug]: fixed

File: client/templates/includes/eventDetails.pug [hunk 3]
```diff
@@ -8,7 +8,9 @@ div(data-hook="event-details")
 
     div.col-md-8.inline(id="event-trigger-expression" data-hook="event-trigger-expression")
```
[the remark this answers is not in the document]
Comment (BryanRumsey) [reply client/templates/includes/eventDetails.pug]: fixed and fixed

**Review by jtcooper10 — APPROVED** [review]
Looks good.

**Pull request merged**